Cargo.toml

```diff
@@ -38,6 +39,7 @@ num-integer = { version = "0.1.36", default-features = false }
 num-traits = { version = "0.2", default-features = false }
 rustc-serialize = { version = "0.3.20", optional = true }
 serde = { version = "1.0.99", default-features = false, optional = true }
+pure-rust-locales = "0.2.1"
```

Comment:
this needs to be `optional = true` and `locals = ["pure-rust-locals"]` above in order for everything to work correctly.

Reply:
done